src/components/modals/WalletModal/index.tsx

```diff
@@ -188,7 +188,11 @@ const WalletModal: React.FC = () => {
       }
     } catch (error) {
       if (error instanceof UnsupportedChainIdError) {
-        activate(connector) // a little janky...can't use setError because the connector isn't set
+        let c = connector
```

Comment:
I think that each variable name should always be written out. It just improves code readability a lot.

Maybe 
```
activate(connector[chainId] instanceof WalletConnectConnector ?connector[chainId], connector)

```
is a better syntax

Reply:
I agree with your suggestion. Thank you for it.